fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/EventListAdapter.java

```diff
@@ -119,6 +121,10 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
       }
       expandIcon.setRotation(expanded[eventCategory.ordinal()] ? 180f : 0f);
 
+      String numberOfEventsInCategory =
+          "(".concat(String.valueOf(eventsMap.get(eventCategory).size())).concat(")");
```

Comment:
Wouldn't it be simpler to write this ?
```suggestion
      String numberOfEventsInCategory = "(" + eventsMap.get(eventCategory).size() + ")";
```